scripts/resolve_model_quota.ps1

```diff
@@ -0,0 +1,102 @@
+param (
+    [string]$Location,
+    [string]$Model,
+    [string]$Format,
+    [string]$DeploymentType,
+    [string]$CapacityEnvVarName,
+    [int]$Capacity
+)
+
+# Verify all required parameters are provided
+$MissingParams = @()
+
+if (-not $Location) {
+    $MissingParams += "location"
+}
+
+if (-not $Model) {
+    $MissingParams += "model"
+}
+
+if (-not $Capacity) {
+    $MissingParams += "capacity"
+}
+
+if (-not $Format) {
+    $MissingParams += "format"
+}
+
+if (-not $DeploymentType) {
+    $MissingParams += "deployment-type"
+}
+
+if ($MissingParams.Count -gt 0) {
+    Write-Error "❌ ERROR: Missing required parameters: $($MissingParams -join ', ')"
+    Write-Host "Usage: .\resolve_model_quota.ps1 -Location <LOCATION> -Model <MODEL> -Format <FORMAT> -Capacity <CAPACITY> -CapacityEnvVarName <ENV_VAR_NAME> [-DeploymentType <DEPLOYMENT_TYPE>]"
+    exit 1
+}
+
+if ($DeploymentType -ne "Standard" -and $DeploymentType -ne "GlobalStandard") {
+    Write-Error "❌ ERROR: Invalid deployment type: $DeploymentType. Allowed values are 'Standard' or 'GlobalStandard'."
+    exit 1
+}
+
+$ModelType = "$Format.$DeploymentType.$Model"
+
+Write-Host "🔍 Checking quota for $ModelType in $Location ..."
+
+# Get model quota information
+$ModelInfo = az cognitiveservices usage list --location $Location --query "[?name.value=='$ModelType']" --output json | ConvertFrom-Json
```

Comment:
We can take First element, to avoid filtering the values by hand:
```
 $ModelInfo = az cognitiveservices usage list --location $Location --query "[?name.value=='$ModelType'] | [0]" --output json | ConvertFrom-Json
```
Then we can just say
```
$ModelInfo.CurrentValue
$ModelInfo.limit
```